model/Core/Classes/Agent.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Agent
+
+## Summary
+
+Agent represents anything acting on a system.
```

Comment:
Suggest changing to "An entity that has the potential to act."

This would align it with the the basic definitions of agent in the W3C PROVO ontology, FIBO ontology, FOAF ontology, and dbpedia ontology.
It also leaves it minimal and flexible for 3.0 as was agreed to.